src/rust/ide/json-rpc/Cargo.toml

```diff
@@ -8,11 +8,12 @@ edition = "2018"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-enso-prelude = { version = "0.1.0"  , path = "../../lib/prelude" }
-shapely      = { version = "0.1.0"  , path = "../../lib/shapely/impl" }
-utils        = { version = "0.1.0"  , path = "../utils" }
-serde        = { version = "1.0", features = ["derive"] }
-futures      = "0.3.1"
-failure      = "0.1.6"
-serde_json   = "1.0"
-shrinkwraprs = "0.3.0"
+enso-prelude      = { version = "0.1.0"  , path = "../../lib/prelude" }
+ensogl-system-web = { version = "0.1.0"  , path = "../../lib/system/web"   }
+shapely           = { version = "0.1.0"  , path = "../../lib/shapely/impl" }
+utils             = { version = "0.1.0"  , path = "../utils" }
+serde             = { version = "1.0", features = ["derive"] }
+futures           = "0.3.1"
+failure           = "0.1.6"
+serde_json        = "1.0"
+shrinkwraprs      = "0.3.0"
```

Comment:
Make the dependency list looking similar to other Cargo.tomls:
* use `{ version = }` for all and align {}
* split to two groups (internal and external) and sort them alphabetically (move `serde`)